Products/zms/zpt/common/zmi_body_header.zpt

```diff
@@ -73,7 +73,7 @@
 					</tal:block>
 					<div class="dropdown-divider"></div> 
 					<a class="dropdown-item" href="manage_zmi_logout" data-turbolinks="false"
-						tal:attributes="href python:'{}?lang={}'.format(here.getConfProperty('ZMS.logout.href',''),request['lang']) if here.getConfProperty('ZMS.logout.href','')!='' else 'manage_zmi_logout';
+						tal:attributes="href python:'{}?lang={}&__ac_name={}'.format(here.getConfProperty('ZMS.logout.href',''),request['lang'],request.get('AUTHENTICATED_USER')) if here.getConfProperty('ZMS.logout.href','')!='' else 'manage_zmi_logout';

```

Comment:
this does not seem to be related to this PR's subject

Reply:
You're right.